react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+export default class StopWatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      start: false,
+      count: 0
+    };
+    this.intervalId = null;
+    this.handleClick = this.handleClick.bind(this);
+    this.resetTimer = this.resetTimer.bind(this);
+  }
+
+  handleClick(e) {
+    e.stopPropagation();
+    this.toggleButton();
+    if (this.state.start === false) {
+      this.stateToggle();
+      this.intervalId = setInterval(() => {
+        this.timer();
+      }, 1000);
+    } else {
+      clearInterval(this.intervalId);
+      this.intervalId = null;
+      this.stateToggle();
+    }
+  }
+
+  toggleButton() {
+    const button = document.querySelector('.fa-solid');
+    if (button.className === 'fa-solid fa-play') {
+      button.className = 'fa-solid fa-pause';
+    } else {
+      button.className = 'fa-solid fa-play';
+    }
+  }
```

Comment:
make sure you update this so you are not interacting with the DOM directly

Reply:
Updated